rust/lib/src/lib.rs

```diff
@@ -54,6 +62,20 @@ fn construct_uri_load_config(
     Ok((config, lightwalletd_uri))
 }
 
+pub fn init_logging() -> String {
+    // this is only for Android
+    #[cfg(target_os = "android")]
```

Comment:
is there a ios version of this function?

Reply:
nope! in IOS the native logging is working without any action.